model/clusters_mgmt/v1/registry_allowlists_resource.model

```diff
@@ -0,0 +1,75 @@
+/*
+Copyright (c) 2024 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Manages the registry allowlists.
+resource RegistryAllowlists {
+	// Retrieves the list of registry allowlists.
+	method List {
+		// Index of the requested page, where one corresponds to the first page.
+		in out Page Integer = 1
+
+		// Number of items contained in the returned page.
+		in out Size Integer = 100
+
+		// Search criteria.
+		//
+		// The syntax of this parameter is similar to the syntax of the _where_ clause of a
+		// SQL statement, but using the names of the attributes of the registry allowlists
+		// instead of the names of the columns of a table. For example, in order to retrieve all
+		// the credentials with a specific cloud provider and creation time the following is required:
```

Comment:
Minor: the comment refers to "the credentials" I guess the example should be focus on RegistryAllowlist model instead.

Reply:
good catch, leftovers, updated them.